lib/src/generator/templates.aot_renderers_for_html.dart

```diff
@@ -2969,7 +2969,7 @@ String _deduplicated_lib_templates_html__head_html(
   buffer.write('''<!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">  <meta charset="utf-8">
```

Comment:
It seems like including all of Font Awesome for two icons might be a lot, it's nice to keep the site download size down.

Since we include it already, could we use Material icons instead? They might match better as well. We might use them in a few places already, at least the [Source button](https://github.com/dart-lang/dartdoc/blob/master/lib/templates/html/_source_link.html#L2).

Reply:
Thank you for the feedback! I went ahead and searched for Material icons and found that it does not have a moon icon atleast what I saw in the [Flutter API](https://api.flutter.dev/flutter/material/Icons-class.html), just the sun icons and both icons would be needed.

Reply:
They are a bit tricky to find, but there are a few options called "Dark Mode" (which corresponds to "Light Mode") and another option called "Nightlight".

https://fonts.google.com has a better search which supports synonyms like "Moon": https://fonts.google.com/icons?icon.set=Material+Icons

Reply:
Thank you! working on changing it currently

Reply:
Changed the icons to the ones I found in fonts.google, thank you for the feedback!

![darkmode-new-icons](https://user-images.githubusercontent.com/106621169/183224773-4a1e9ab6-3b7b-4e65-8ef3-27c104506ddb.png)

![lightmode-new-icons](https://user-images.githubusercontent.com/106621169/183224774-81f562e0-0ac3-47d1-bf4b-1a84bd13d7c1.png)